patches/chromium/worker_context_will_destroy.patch

```diff
@@ -23,7 +23,7 @@ index 784e0a5166d6bece3f6a251e7cab1192e576af22..efd8ef49f1f2b6ecd6c3638b56e40c4e
 +      v8::Local<v8::Context> context) {}
 +
    // Overwrites the given URL to use an HTML5 embed if possible.
-   // An empty URL is returned if the URL is not overriden.
+   // An empty URL is returned if the URL is not overridden.
```

Comment:
This typo belongs to Chromium's file `content/public/renderer/content_renderer_client.h` that is being patched here. We need to leave the spelling alone in order for the patch to succeed.